From 31382a85e7ddf504a785e561c4accf2fc09ab8cc Mon Sep 17 00:00:00 2001
From: "Tao, Jing" <jing.tao@intel.com>
Date: Thu, 4 Sep 2014 11:38:19 +0800
Subject: [PATCH 093/117] mt9m114: elminiate the debug info print

The regular printing for debug slows down the system a lot.

Using dev_dbg for debut printing.

Change-Id: I3542df8450e4844302976344081247fe58f49037
Signed-off-by: Tao, Jing <jing.tao@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c b/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c
index 12ba009..ffb89f2 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c
+++ b/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c
@@ -1099,7 +1099,9 @@ static long mt9m114_s_exposure(struct v4l2_subdev *sd,
     u16 exposure_local[3];
     u32 RegSwResetData = 0;
 
-    dev_err(&client->dev, "%s(0x%X 0x%X 0x%X)\n", __func__, exposure->integration_time[0], exposure->gain[0], exposure->gain[1]);
+    dev_dbg(&client->dev, "%s(0x%X 0x%X 0x%X)\n", __func__,
+		    exposure->integration_time[0], exposure->gain[0],
+		    exposure->gain[1]);
 
     coarse_integration = exposure->integration_time[0];
 //    fine_integration = ExposureTime.FineIntegrationTime;
-- 
1.9.3

